debian/changelog

```diff
@@ -1,3 +1,19 @@
+opx-nas-l3 (3.9.0) unstable; urgency=medium
+
+  * Update: Added UT script for PBR ACL cleanup dependency issue
+  * Bugfix: 'show ip arp summary' and 'ipv6 neighbor summary' do not clear the 
+            count on shutting down the ICL ports.
+  * Bugfix: arp mismatch is seen between vlt peers after a node reload on a TOR switch.
```

Comment:
Remove mention of fixes having to do with VLT since they are not relevant to OPX by itself